src/Temperature_LM75_Derived.h

```diff
@@ -265,6 +265,36 @@ class Generic_LM75_9_to_12Bit : public Generic_LM75_9_to_12Bit_Compatible {
     : Generic_LM75_9_to_12Bit_Compatible(&Wire, i2c_address, &Generic_LM75_12Bit_Attributes) { };
 };
 
+class OnSemi_OneShot : public Generic_LM75_12Bit {
```

Comment:
I would suggest naming this based on the model rather than the feature:

```suggestion
class OnSemi_NCTx75 : public Generic_LM75_12Bit {
```